content/docs/user-guide/deploying/kubernetes.md

```diff
@@ -12,6 +21,344 @@ pip install mlem[kubernetes]
+service created. status='{'conditions': None, 'load_balancer': {'ingress': None}}'
+✅  Deployment ml is up in mlem namespace
+```
+
+where:
+
+- `service_name` is a name of one's own choice, of which corresponding
+  `service_name.mlem` and `service_name.mlem.state` files will be created.
+- `model` denotes the path to model saved via `mlem`.
+- `service_type` is configurable and is passed as `loadbalancer`. The default
+  value is `nodeport` if not passed.
+
+### Checking the docker images
+
+One can check the docker image built via `docker image ls` which should give the
+following output:
+
+```
+REPOSITORY                                TAG                                IMAGE ID       CREATED         SIZE
+ml                                        4ee45dc33804b58ee2c7f2f6be447cda   16cf3d92492f   3 minutes ago   778MB
+...
+```
+
+### Checking the kubernetes resources
+
+Pods created can be checked via `kubectl get pods -A` which should have a pod in
+the `mlem` namespace present as shown below:
+
+```
+NAMESPACE     NAME                               READY   STATUS    RESTARTS        AGE
+kube-system   coredns-6d4b75cb6d-xp68b           1/1     Running   7 (12m ago)     7d22h
+...
+kube-system   storage-provisioner                1/1     Running   59 (11m ago)    54d
+mlem          ml-cddbcc89b-zkfhx                 1/1     Running   0               5m58s
+```
+
+By default, all resources are created in the `mlem` namespace. This ofcourse is
+configurable using `--conf namespace=prod` where `prod` is the desired namespace
+name.
+
+### Making predictions via mlem
+
+One can of course use the
+[`mlem deployment apply`](/doc/command-reference/deployment/apply) command to
+ping the deployed endpoint to get the predictions back. An example could be:
+
+```cli
+$ mlem deployment apply service_name data --json
+
+[0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2, 2]
+```
+
+where `data` is the dataset saved via `mlem`.
+
+### Deleting the Kubernetes resources
+
+A model can easily be undeployed using `mlem deploy remove service_name` which
+will delete the `pods`, `services` and the `namespace` i.e. clear the resources
+from the cluster. The docker image will still persist in the registry though.
+
+<details>
+
+### ⚙️ About which cluster to use
+
+MLEM tries to find the kubeconfig file from the environment variable
+`KUBECONFIG` or the default location `~/.kube/config`.
+
+If you need to use another path, one can pass it with
+
+`--conf kube_config_file_path=...`
+
+</details>
+
+## Case Study: Using EKS cluster with ECR on AWS
```

Comment:
Nice part! This could be a separate page (How to? Tutorial?) maybe? Or better to leave it here? WDYT @jorgeorpinel

Reply:
Seems like an example of all the info above. It's kind of repetitive, maybe use this platform throughout the docs as the default example instead? Or extract to a blog post. For now it could  be moved to a sub-page of this one i.e. in path /doc/user-guide/deploying/kubernetes/ekr or something like that.

Reply:
> p.s. try to link to 3rd party tool docs for context in general. In this case e.g. to https://docs.aws.amazon.com/eks/latest/userguide/what-is-eks.html . Same for ECR

Reply:
Ok, I think renaming to "Example" is a good option, to keep the same structure of UG pages.

Reply:
cc @madhur-tandon - I think you can start processing the feedback here.

Reply:
I have renamed it to `Example` and moved it to bottom for now. Do we need a separate page for this? Or moving it to the bottom is fine for now?

Also, I have added links for `EKS` and `ECR`

Reply:
Don't see the changes in the preview deployment, but should be ok for now. Thanks!

Reply:
Yes, I am yet to push, will do at once.

Reply:
It looks better in https://github.com/iterative/mlem.ai/pull/188/files#diff-2725212d8a5f135e8dcd3b414cc2e8f9f57fbbb0cee7f756eecdb70698cc4f0d 👍🏼